nornir/plugins/tasks/networking/napalm_ping.py

```diff
@@ -0,0 +1,55 @@
+from typing import Optional
+from nornir.core.task import Result, Task
+
+
+def napalm_ping(
+    task: Task,
+    dest: str,
+    source: Optional[str] = None,
+    ttl: Optional[int] = 255,
+    timeout: Optional[int] = 2,
+    size: Optional[int] = 100,
+    count: Optional[int] = 5,
+    vrf: Optional[str] = None,
+) -> Result:
+    """
+    Executes ping on the device and returns a dictionary with the result.
+
+    Arguments:
+      dest(destination) – Host or IP Address of the destination. (type=string)
```

Comment:
Can you format this consistently with the other plugins? For instance:

https://github.com/nornir-automation/nornir/blob/develop/nornir/plugins/tasks/networking/netmiko_save_config.py

Reply:
@dbarrosop  i have made changes according to format.